pkg/kubelet/kubelet_pods.go

```diff
@@ -117,6 +117,12 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 	mountEtcHostsFile := (pod.Spec.SecurityContext == nil || !pod.Spec.HostNetwork) && len(podIP) > 0 && runtime.GOOS != "windows"
 	glog.V(3).Infof("container: %v/%v/%v podIP: %q creating hosts mount: %v", pod.Namespace, pod.Name, container.Name, podIP, mountEtcHostsFile)
 	mounts := []kubecontainer.Mount{}
+
+	propagations, err := v1helper.GetPodPropagation(pod.Annotations)
```

Comment:
i am wondering if the kubelet should reject pods that use this annotation during admission if the feature is not enabled.

i know we said we would deprecate the annotation, but i dont always trust our future self.